site/utils/oauth.mjs

```diff
@@ -0,0 +1,6 @@
+
+import {Client} from 'oceanic.js'
```

Comment:
OAuth should be done with something internal (I use https://pastebin.com/CsgakMjL), no reason to use a bloated external package

Reply:
The code you linked does not have any rate limit support, oceanic does. with that being said oceanic is bloated for this usecase and i will be looking into a better way to implement oauth.